Cargo.toml

```diff
@@ -27,18 +27,20 @@ regex = "1.6.0"
 
 #TODO: Investigate rweb to use openapi spec for free
 # https://github.com/kdy1/rweb
-actix-files = "0.6.2"
-actix-web = "4.1.0"
-actix-service = "2.0.2"
-serde = { version = "1.0.140", features = ["derive"] }
-serde_json = "1.0.82"
+actix-files = "0.6"
+actix-web = "4"
+actix-web-validator = "5"
+actix-service = "2"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+validator = { version = "0.16", features = ["derive"] }
```

Comment:
Should we stick to a minor version for these libs?

Reply:
That's not necessary, minor versions does not break apis, just bug fixes and improvements